kiva/trait_defs/ui/wx/kiva_font_editor.py

```diff
@@ -119,7 +119,13 @@ def str_font ( self, font ):
     def all_facenames(self):
         """ Returns a list of all available font typeface names.
         """
-        return sorted({f.name for f in fontManager.ttflist})
+        return _all_facenames()
+
+
+def _all_facenames():
+    """ Return a list of all available (TTF) font typeface names."""
+    font_manager = default_font_manager()
+    return sorted({f.name for f in font_manager.ttflist})
```

Comment:
I'm curious why you opted to break this out into a function?

Reply:
I was lazy when I tried to put up tests for all uses of font managers before the refactoring and jumped into the conclusion that it would be hard to test the editor factory instance... I think I can move this back.